react/features/polls/components/PollDialog.js

```diff
@@ -0,0 +1,406 @@
+// @flow
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import v1 from 'uuid/v1';
+
+import { DialogWithTabs, hideDialog } from '../../base/dialog';
+import { translate } from '../../base/i18n';
+import { getLocalParticipant } from '../../base/participants';
+
+import {
+    startPoll,
+    vote, endPoll
+} from '../actions';
+import { getUniquePollChoices } from '../functions';
+
+import PollCreateForm from './PollCreateForm';
+import PollResultsForm from './PollResultsForm';
+import VoteForm from './VoteForm';
+
+type Props = {
+
+    /**
+     * Redux dispatch method.
+     */
+    dispatch: Function,
+
+    /**
+     * True if there is an active poll session now.
+     */
+    isPollRunning: boolean,
```

Comment:
I do check in lib-jitsi-meet but one of the created poll must be fully created -I mean other participant must hear from him- before my checks can work. Otherwise this will case inconsistency